### PR TITLE
Feature: Add option to return API response or dowload labels on checkout

### PR DESCRIPTION
**Pull request by dahoo**

### CODE REVIEW

**Review by nhoffmann — CHANGES_REQUESTED** [review]
No test?

File: spec/office_spec.rb [hunk 1]
```diff
@@ -17,32 +17,54 @@
     expect(office.user.wallet_balance).to be > 0
   end
 
-  it 'should order national post stamps' do
-    office = Office.new
+  describe "checkout!" do
```
Comment (nhoffmann) [inline spec/office_spec.rb]: It seems like the project uses single quotes.

File: spec/office_spec.rb [hunk 2]
```diff
@@ -17,32 +17,54 @@
...
+
+      it 'should order national post stamps' do
+        expect(labels.length).to be >= 100
+      end
+    end
 
-    labels = office.checkout!(page_format: 25)
-    expect(labels.length).to be >= 100
+    context "when not downloading labels" do
+      let(:labels) { office.checkout!(page_format: 25, download_labels: false) }
```
Comment (nhoffmann) [inline spec/office_spec.rb]: It feels weird to name the response `labels`, when it is an api response.

**Review by nhoffmann — APPROVED** [review]
Some minor style issues, LGTMO

**Comment by dahoo** [comment]
[Als Admin möchte ich die API der Post nutzen, um direkt Briefmarken zu erzeugen](https://trello.com/c/EmeyEuqI/2034-als-admin-m%C3%B6chte-ich-die-api-der-post-nutzen-um-direkt-briefmarken-zu-erzeugen)

**Pull request merged**